test/rekt/features/trigger/feature.go

```diff
@@ -54,7 +54,7 @@ func TriggerDependencyAnnotation() *feature.Feature {
 	f.Setup("broker is addressable", broker.IsAddressable(brokerName))
 
 	psourcename := "test-ping-source-annotation"
-	dependencyAnnotation := `{"kind":"PingSource","name":"test-ping-source-annotation","apiVersion":"sources.knative.dev/v1"}`
+	dependencyAnnotation := `'{"kind":"PingSource","name":"test-ping-source-annotation","apiVersion":"sources.knative.dev/v1"}'`
```

Comment:
I think we can fix it in the template by adding quotes to the value https://github.com/knative/eventing/blob/96e5c1853b7ce5af3fb934155f7e4c1d3454bcba/test/rekt/resources/trigger/trigger.yaml#L22-L24

and catch any future value/variable?

Reply:
Oh yeah, great point!